js/index.js

```diff
@@ -1,61 +1,57 @@
-let listBooks = [];
+/* eslint-disable no-undef */
+
+// let listBooks
+const listBooks = new Library();
 // Local Storage
 const saveToLocalStorage = () => {
-  localStorage.setItem('Library', JSON.stringify(listBooks));
+  localStorage.setItem('MY-Library', JSON.stringify(listBooks));
 };
 
 const getDataFromLocalStorage = () => {
-  const data = JSON.parse(localStorage.getItem('Library'));
+  const data = JSON.parse(localStorage.getItem('MY-Library'));
   if (data !== null) {
-    listBooks = data;
+    listBooks.setBooks(data.books);
+  } else {
+    saveToLocalStorage();
   }
 };
 
 function displayBooks() {
   const section = document.querySelector('#book-list');
-  let books = '';
   getDataFromLocalStorage();
-
-  listBooks.forEach((book, index) => {
-    books += `<article class="book">
-    <p>${book.title} <br> ${book.author}</p>
-    <button type="button" id="${index}" class="remove-btn" onclick="removeBook(${index})">Remove</button>
-    <hr>
-  </article>`;
-  });
-
-  if (listBooks.length === 0) {
-    books = '<p>Library is empty...</p>';
-  }
+  const books = listBooks.getBooks();
   section.innerHTML = books;
 }
 
 function removeBook(id) {
```

Comment:
- [ ] Great job so far the add and remove is working fine 👍🏻  however as requested by the previous reviewer, You must only use JavaScript `classes` instead of `objects` and `arrays`, and note that you can use objects and arrays but it has to be **inside** the classes. actually, I can see that you still have javascript `functions` in your index.js file, Please remove them.

Reply:
Hi Ben, We did it already. I refactor the code. Please verify. 
![image](https://user-images.githubusercontent.com/39686386/211702030-6ae74053-c954-4dd9-aa26-7b353aa2b690.png)